hedera-node/hedera-consensus-service-impl/src/test/java/com/hedera/node/app/service/consensus/impl/test/handlers/ConsensusCreateTopicHandlerTest.java

```diff
@@ -308,9 +301,9 @@ void translatesInvalidExpiryException() {
         given(expiryValidator.resolveCreationAttempt(anyBoolean(), any()))
                 .willThrow(new HandleException(ResponseCodeEnum.INVALID_EXPIRATION_TIME));
 
-        final var failure = assertThrows(
+        final var msg = assertThrows(
```

Comment:
Not sure why this was renamed to `msg` instead of `failure`. The `assertThrows` method returns an exception, not a string/message

Reply:
Just to be consistent with all other code. msg or failure is just a var name.